### ISSUE TITLE
Invalid import for dsl/mockService

### ISSUE THREAD

**Issue by cascornelissen** [pact-foundation/pact-js#911]
### Software versions

- **OS**: N/A
- **Consumer Pact library**: 10.0.1
- **Provider Pact library**: N/A
- **Node Version**: N/A

### Issue Checklist

Please confirm the following:

- [x] I have upgraded to the latest
- [x] I have the read the FAQs in the Readme
- [x] I have triple checked, that there are **no unhandled promises** in my code and have [read](https://github.com/pact-foundation/pact-js/#test-fails-when-it-should-pass) the section on intermittent test failures
- [ ] I have set my log level to debug and attached a log file showing the complete request/response cycle
- [ ] For bonus points and virtual high fives, I have created a reproduceable git repository (see below) to illustrate the problem

### Expected behaviour
Imports resolve correctly

### Actual behaviour
The following error is emitted

```
node_modules/@pact-foundation/pact/src/httpPact/index.d.ts:3:29 - error TS2307: Cannot find module 'dsl/mockService' or its corresponding type declarations.

    3 import { MockService } from 'dsl/mockService';
```

### Steps to reproduce

Install `10.0.1` in a TypeScript project and try to build including emitting of declarations.

### Relevant log files
N/A

---

This was fixed in https://github.com/pact-foundation/pact-js/commit/7ed342ed0bce331b7e01a6eb41d41e9228ae5b5f already and I've been hoping for a release the past few days but now this is really blocking us so I'm wondering when a new release will come out?

**Comment by YOU54F** [comment]
Just kicked off a release workflow now

**Comment by YOU54F** [comment]
@cascornelissen released as 10.0.2

**Comment by cascornelissen** [comment]
Thanks for the quick reply, much appreciated ❤️
